tests/misc/test_basic_without_ssh.py

```diff
@@ -25,21 +25,25 @@ def existing_shared_sr(host):
     assert sr is not None, "A shared SR on the pool is required"
     return sr
 
-@pytest.mark.incremental # tests depend on each other. If one test fails, don't execute the others
 @pytest.mark.multi_vms # run them on a variety of VMs
 @pytest.mark.big_vm # and also on a really big VM ideally
-class TestBasicNoSSH:
-    def test_start(self, imported_vm):
-        vm = imported_vm
-        # if VM already running, stop it
-        if (vm.is_running()):
-            logging.info("VM already running, shutting it down first")
-            vm.shutdown(verify=True)
-        vm.start()
-        # this also tests the guest tools at the same time since they are used
-        # for retrieving the IP address and management agent status.
-        vm.wait_for_os_booted()
+def test_vm_start_stop(imported_vm):
+    vm = imported_vm
+    # if VM already running, stop it
+    if (vm.is_running()):
+        logging.info("VM already running, shutting it down first")
+        vm.shutdown(verify=True)
+    vm.start()
+    # this also tests the guest tools at the same time since they are used
+    # for retrieving the IP address and management agent status.
+    vm.wait_for_os_booted()
+
+    vm.shutdown(verify=True)
 
+@pytest.mark.multi_vms # run them on a variety of VMs
+@pytest.mark.big_vm # and also on a really big VM ideally
+@pytest.mark.usefixtures("running_vm")
```

Comment:
I avoided suggesting the use of this fixture, because this fixture checks that the VM answers to SSH, which contradicts the requirements in this test file.

Reply:
splitting part of `running_vm` into reusable `started_vm`